.github/workflows/reference-blockchain-tests.yml

```diff
@@ -75,7 +75,6 @@ jobs:
         timeout-minutes: 180
         continue-on-error: true
         env:
-          REFERENCE_TESTS_PARALLELISM: 20
```

Comment:
Some tests consume a lot of heap space so it is not feasible to have tests run in parallel as the sum of heap usage of all concurrent tests will exceed the max heap size.